From 62bb5f36abf473e096ac4d38bc4059dd3eef51ea Mon Sep 17 00:00:00 2001
From: Gerd Hoffmann <kraxel@redhat.com>
Date: Thu, 23 Jun 2011 12:41:46 -0300
Subject: [RHEL6 qemu-kvm PATCH 071/115] usb: Add Interface Association Descriptor descriptor type

RH-Author: Gerd Hoffmann <kraxel@redhat.com>
Message-id: <1308832951-8995-71-git-send-email-kraxel@redhat.com>
Patchwork-id: 28393
O-Subject: [RHEL-6.2 kvm PATCH 070/115] usb: Add Interface Association Descriptor descriptor type
Bugzilla: 561414 632299 645351 711354
RH-Acked-by: Hans de Goede <hdegoede@redhat.com>
RH-Acked-by: Jes Sorensen <Jes.Sorensen@redhat.com>
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>
RH-Acked-by: Kevin Wolf <kwolf@redhat.com>

From: Brad Hards <bradh@frogmouth.net>

Signed-off-by: Brad Hards <bradh@frogmouth.net>
Signed-off-by: Gerd Hoffmann <kraxel@redhat.com>
(cherry picked from commit c6d3ad0fad34b33557976e579ab0e4159989506f)
---
 hw/usb.h |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

Signed-off-by: Eduardo Habkost <ehabkost@redhat.com>
---
 hw/usb.h |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/hw/usb.h b/hw/usb.h
index aebd0b9..c7e52dd 100644
--- a/hw/usb.h
+++ b/hw/usb.h
@@ -120,6 +120,7 @@
 #define USB_DT_ENDPOINT			0x05
 #define USB_DT_DEVICE_QUALIFIER         0x06
 #define USB_DT_OTHER_SPEED_CONFIG       0x07
+#define USB_DT_INTERFACE_ASSOC          0x0B
 
 #define USB_ENDPOINT_XFER_CONTROL	0
 #define USB_ENDPOINT_XFER_ISOC		1
-- 
1.7.3.2

